Instagram/Instagram/MenuCollectionViewCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MenuCollectionViewCell.swift
+//  Instagram
+//
+//  Created by Edvard on 23.07.2021.
+//
+
+import UIKit
+
+class MenuCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var labelName: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var nameLabel: UILabel!
```
6 й раз пишу тип в конце имени пишем!